symfony-cmf/routing-bundle/2.0/config/packages/cmf_routing_bundle.yaml

```diff
@@ -0,0 +1,14 @@
+# Read the documentation: https://symfony.com/doc/master/cmf/bundles/routing/configuration.html
+cmf_routing:
+    chain:
+        routers_by_id:
+            cmf_routing.dynamic_router: 200
+            router.default:             100
+    dynamic:
+        persistence:
+            phpcr:
+                enabled: true
```

Comment:
We can't enable it by default, cause we should have installed the complete phpcr stack then before.